docs/TESTS.md

```diff
@@ -1,15 +1,114 @@
 # Tests
 
-<!-- TODO: write document
+This track uses the [unitt unit-testing framework][unitt].
 
-  This document should describe everything related to running tests in the track.
+Each exercise will have a similar structure as seen below:
 
-  If your track uses skipped tests, this document can explain why thet is used and
-  how to unskip tests.
+```
+HELP.md
+README.md
+src
+tester.art
+tests
+```
 
-  This document can also link to the testing framework documentation.
+To run the exercise's test suite on the CLI, run `arturo tester.art` (or `exercism test`) inside the exercise folder.
 
-  The contents of this document are displayed on the track's documentation
-  page at `https://exercism.org/docs/tracks/<track>/tests`.
+## Structure
 
-  See https://exercism.org/docs/building/tracks/docs for more information. -->
+Your solution goes inside `src\<slug>.art` where `slug` is the exercise slug.
+For the Leap exercise, this is `leap` so your solution goes into `src\leap.art`.
+
+`tester.art` is what Unitt calls a runner which is responsible for locating, finding, and running the tests for each exercise.
+The runner looks inside the `tests` folder to find the test file at `tests\test-<slug>.art` or `tests\test-leap.art` as an example.
+
+This test file contains one or more test suites, noted by the `suite` word and grouping similar tests, each noted by the `test` word.
+The first test is run every time, but subsequent tests are marked with a `skip` attribute.
+To run a skipped test, change `test.skip` to `test` for that test and rerun the test suite as documented above.
+Once all tests pass locally, you can submit your solution with `exercism submit` inside the exercise folder.
+
+## Test Output
+
+Let's use the following example test file to look at the Unitt test output from running `tester.art`.
+
+```arturo
+import {unitt}!
+import {src/leap}!
+
+suite "Leap" [
+    test "a year not divisible by 4 is a common year" [
+        result: isLeap? 2015
+        assert -> false = result
+    ]
+
+    test.skip "a year divisible by 2 and not divisible by 4 is a common year" [
+        result: isLeap? 1970
+        assert -> false = result
+    ]
+]
+```
+
+If we use the starting code in `src\leap.art` without modification,
```

Comment:
If this is not a Windows OS based language, should we keep the path designations with forward slashes?